conda_build/environ.py

```diff
@@ -212,6 +213,25 @@ def get_git_info(git_exe, repo, debug):
         log.debug("Error obtaining git commit information.  Error was: ")
         log.debug(str(error))
 
+    try:
+        # get the timestamp
+        output = utils.check_output_env(
+            [git_exe, "show", "-s", "--format=%ci", "HEAD"],
```

Comment:
`--format=%cI` (capital I) will give you strict ISO 8601, `%ci` is an "ISO 8601-like format"